ownCloudTests/File List/Subclasses/MockOCCore.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  MockOCCore.swift
+//  ownCloudTests
+//
+//  Created by Javier Gonzalez on 10/01/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+import ownCloudSDK
+
+class MockOCCore: OCCore {
+
+	var query:MockOCQuery?
```

Comment:
If you don't set `query` from outside with a `nil` value, you don't need this to be optional, the code would look like:
```Swift

        var query: MockOCQuery

	convenience init(query: MockOCQuery, bookmark: OCBookmark, issue: OCIssue?) {
                self.query = query
		self.init(bookmark: bookmark)
		self.issue = issue
	}
```